src/test/kotlin/com/aol/mobile/sdk/apitracker/DescriptorsDeserializerTest.kt

```diff
@@ -21,7 +21,7 @@ class DescriptorsDeserializerTest {
                   "public",
                   "final"
                 ],
-                "name": "com.aol.mobile.sdk.player.OneSDK",
```

Comment:
Why this file is changed?

Reply:
> Why this file is changed?

This file has links to classes which namespace was changed.